packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -89,19 +91,14 @@ export class EditorPage {
 	 *
 	 * @returns {Promise<Frame>} iframe holding the editor.
 	 */
-	async waitUntilLoaded(): Promise< Frame > {
-		const frame = await this.getEditorFrame();
-		// Traditionally we try to avoid waits not related to the current flow.
-		// However, we need a stable way to identify loading being done. NetworkIdle
-		// takes too long here, so the most reliable alternative is the title being
-		// visible.
-		const titleLocator = this.frameLocator.locator( selectors.editorTitle );
-		await titleLocator.waitFor();
+	async waitUntilLoaded(): Promise< void > {
```

Comment:
Change the signature of `waitUntilLoaded` to explicitly only perform what the method name states.
This way we can enforce the use of `getEditorFrame` when the caller requires the frame object itself.